t/t5403-post-checkout-hook.sh

```diff
@@ -17,6 +17,11 @@ test_expect_success setup '
 	git clone ./. clone1 &&
 	git clone ./. clone2 &&
 	( cd clone2 &&
+	git checkout -b rebase-on-me &&
+	echo Data for commit2. > c &&
+	git add c &&
+	git commit -m rebase-on-me &&
```

Comment:
How about using `test_commit` here? Not only is it more concise, it also generates a stable commit, i.e. the same commit every time the test script is run.

Reply:
Done